lambda.tf

```diff
@@ -0,0 +1,63 @@
+locals {
+  code_version  = var.autoscaler_version
```

Comment:
It does not seem necessary since it is used only once and uses the variable value as-is.

Reply:
Done